mirrord/layer/tests/sip.rs

```diff
@@ -35,7 +37,27 @@ async fn tmp_dir_read_locally(dylib_path: &Path) {
     let mut layer_connection = LayerConnection::get_initialized_connection(&listener).await;
     println!("Application subscribed to port, sending tcp messages.");
 
-    layer_connection.expect_gethostname(1).await;
+    layer_connection
+        .expect_file_open_for_reading("/etc/hostname", 1)
+        .await;
```

Comment:
so we dont need the `expect_gethostname` anymore? what was the issue on this one?

Reply:
Sometimes we don't get the close message because the process exits before we manage to send the message.

Reply:
got it